scss/_reformat.scss

```diff
@@ -1,9 +1,11 @@
 // Show link source
 // For fully qualified URLs only
-a[href^="http://"], &[href^="https://"], &[href^="ftp://"] {
-	&:after {
-		content: " (" attr(href) ")";
-		font-size: 80%;
+a {
+	[href^="http://"], [href^="https://"], [href^="ftp://"] {
```

Comment:
thank you, I fixed that now in [https://github.com/BafS/Gutenberg/pull/3/commits/83979ceaaa435d04392b60d0013f447e63cc161b](https://github.com/BafS/Gutenberg/pull/3/commits/83979ceaaa435d04392b60d0013f447e63cc161b)